tests/jcli/address/single.rs

```diff
@@ -85,9 +132,10 @@ pub fn test_delegation_address_made_of_random_string() {
 
     // Assertion changed due to issue #306. After fix please change it to correct one
     process_assert::assert_process_failed_and_contains_message(
-        jcli_wrapper::jcli_commands::get_address_delegation_command_default(
+        jcli_wrapper::jcli_commands::get_address_delegation_command(
             &public_key,
             &delegation_key,
+            true,
```

Comment:
The boolean parameter is not informative. Invent an enum with descriptive values?